packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames';
+import Div100vhContainer from '../div100vh-container';
+import FadeWrapper from '../fade-wrapper';
+import PageOverlay from '../page-overlay';
```

Comment:
```suggestion
import PageOverlay from '../page-overlay/page-overlay.jsx';
```

To prevent style duplicates since importing `page-overlay` via index will also import the `stylesheet` file - `page-overlay.scss`. 
For `FadeWrapper` and `Div100vhContainer` it's unnecessary since those only have dummy stylesheet files.